src/main/java/qna/domain/Answer.java

```diff
@@ -32,6 +47,10 @@ public Answer(Long id, User writer, Question question, String contents) {
         this.contents = contents;
     }
 
+    public Answer() {
```

Comment:
기본생성자를 public으로 열어둘 필요는 없습니다.
```suggestion
    protected Answer() {
```

Reply:
다른 엔티티의 생성자도 같이 점검해보아요

Reply:
@Entity 애노테이션을 달아주니 오류가 나서 생각없이 public으로 생성해주었습니다.
아무값도 전달하지 않는 생성자는 굳이 필요가 없을것 같군요.